frontend/packages/ux-editor/src/components/Properties/PageConfigPanel/PageConfigPanel.test.tsx

```
@@ -92,32 +78,33 @@
     expect(uniqueIds).not.toBeInTheDocument();
   });
 
-  it('should not show warning modal when there are no duplicated ids across layouts', () => {
-    renderPageConfigPanel();
+  it('should not show warning modal when there are no duplicated ids across layouts', async () => {
+    await act(async () => {
```

ESLint

[error] Avoid wrapping Testing Library util calls in `act`

(testing-library/no-unnecessary-act)